README.rst

```diff
@@ -133,16 +133,16 @@ Without tox
 
 .. code-block:: bash
 
-    $ pip install -r conans/requirements.txt
-    $ pip install -r conans/requirements_server.txt
-    $ pip install -r conans/requirements_dev.txt
+    $ python -m pip install -r conans/requirements.txt
+    $ python -m pip install -r conans/requirements_server.txt
+    $ python -m pip install -r conans/requirements_dev.txt
 
 
 Only in OSX:
 
 .. code-block:: bash
 
-    $ pip install -r conans/requirements_osx.txt # You can omit this one if not running OSX
+    $ python -m pip install -r conans/requirements_osx.txt # You can omit this one if not running OSX
```

Comment:
This has been removed and made conditional in requirements.txt, can be removed

Reply:
removed completely